packages/gatsby-source-wordpress/jest.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
Can we use the original jest.config? or at least require and override
```
module.exports = {
  ...require('../../jest.config.js'),
  transform: {
    "^.+\\.[jt]sx?$": `<rootDir>/jest-transformer.js`,
  },
  moduleNameMapper: {
    "~/(.*)": `<rootDir>/plugin/src/$1`,
  },
  testPathIgnorePatterns: [`node_modules`, `\\.cache`, `<rootDir>.*/public`],
  transformIgnorePatterns: [`node_modules/(?!(gatsby)/)`],
  globals: {
    __PATH_PREFIX__: ``,
  },
  testURL: `http://localhost`,
  setupFiles: [`<rootDir>/loadershim.js`],
  globalSetup: `./test-site/test-utils/global-setup-jest.js`,
  globalTeardown: `./test-site/test-utils/global-teardown-jest.js`,
  setupFilesAfterEnv: [`./test-site/test-utils/jest.setup.js`],
}
```

Reply:
For sure 👍 I'll just remove jest-related files for now since I removed the tests temporarily